scripts/release_scripts/deploy.py

```diff
@@ -60,6 +59,8 @@
 _PY_GITHUB_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'PyGithub-1.43.7')
 sys.path.insert(0, _PY_GITHUB_PATH)
 
+import github  # isort:skip pylint: disable=wrong-import-position
```

Comment:
Here and in all other files this PR touches: please add a clear comment explaining why the github import needs to be on this line.

This is particularly important in the files which don't have a sys.path.insert() just above it.

Reply:
I think I'll still merge this though, to unbreak. Please do this in a follow-up issue.

Reply:
Okay, I will create a follow-up PR to add a description of why we need github import below all other imports.